omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-nginx-run.erb

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 exec 2>&1
-exec /opt/opscode/embedded/bin/veil-env-helper -o DATA_COLLECTOR_TOKEN=data_collector.token -s REDIS_PASSWORD=redis_lb.password -- chpst -P /opt/opscode/embedded/sbin/nginx -c <%= File.join(node['private_chef']['nginx']['dir'], "etc", "nginx.conf") %>
```

Comment:
[nit] I know it's been like that before, but I think this file could be helped by dropping some `\ \n` in there to make it less unreadable... 💭